resource/lookup_pipeline.yaml

```diff
@@ -21,14 +21,15 @@ jobs:
         type: docker-image
         source:
           repository: quay.io/mojanalytics/lookup-tables-etl
-          tag: v0.0.1
```

Comment:
sure you want to leave this on :latest?

Reply:
Yeah for now anyway. Can change it later if it starts causing problems.